includes/admin/class-wc-payments-admin.php

```diff
@@ -540,6 +533,12 @@ public function add_menu_notification_badge() {
 			return;
 		}
 
+		// If plugin activation date is less than 7 days, do not show the badge.
+		$past_7_days = time() - get_option( 'wcpay_activation_timestamp', 0 ) >= 86400 * 7;
```

Comment:
WP actually [has a nifty constant called `WEEK_IN_SECONDS`](https://github.com/WordPress/wordpress-develop/blob/3c57ccf1b4e0b39ec2aa4c954fd732f1bc96c065/src/wp-includes/default-constants.php#L141) that you could use here!

Reply:
Thank you! I will update it.